data-reconciliation-app/docs/adr/02-data-schema-strategy.md

```diff
@@ -69,13 +69,13 @@ POST /submit-data
 }
 ```
 
-This option requires a member to reset their previously submitted data and resubmit again when they change their schema.
+This option requires a member to reset their previously submitted data and re-submit it when they change their schema.
 
-While this option provides great flexibility for members, it introduces lots of data mapping and validation challenges which will not be feasible within the project timeline
+While this option provides great flexibility for members, it introduces lots of data mapping and validation challenges that won't be feasible within the project timeline.
 
 ### Option3: Defining Data Schema via Deployment
 
-In this strategy, a new data schema is defined at the code level and deployed into CCF network. This also includes implementing data schema validation at the endpoint, service and repository layers before the deployment. It can produce a breaking change or backwards compatible change depending on the changes. This will require all members update their integrations to submit data and to produce reconciliation reports when a new version of the application is deployed. However, data will be consistent end to end from the endpoints all the way to the reporting.
+In this strategy, a new data schema is defined at the code level and deployed into the CCF network. This also includes implementing data schema validation at the endpoint, service, and repository layers before the deployment. It can produce a breaking change or a backward compatible change, depending on the changes. This will require all members to update their integrations to submit data and produce reconciliation reports when a new version of the application is deployed. However, the data will be consistent throughout, from the endpoints all the way to the reporting.
```

Comment:
```suggestion
In this strategy, a new data schema is defined at the code level and deployed into the CCF network. This also includes implementing data schema validation at the endpoint, service, and repository layers before deployment. It can produce a breaking change or a backward compatible change, depending on the changes. This will require all members to update their integrations to submit data and produce reconciliation reports when a new version of the application is deployed. However, the data will be consistent throughout, from the endpoints all the way to the reporting.
```